backend/app/engine/octant_rewards/leftover/with_ppf.py

```diff
@@ -0,0 +1,12 @@
+from app.engine.octant_rewards.leftover import LeftoverPayload, Leftover
+
+
+class LeftoverWithPPF(Leftover):
+    def calculate_leftover(self, payload: LeftoverPayload) -> int:
+        return (
+            payload.staking_proceeds
+            - payload.operational_cost
+            - payload.community_fund
+            - int(payload.ppf / 2)
```

Comment:
I would call it `extra_individual_rewards` how it's here in the doc: https://www.notion.so/octantapp/Octant-Rewards-Domain-Language-36cafb5c95074fef8068e2e3de9ea061
It's not critical but can help as avoid confusion like in the past.

Reply:
ok